site/docs/v1/tech/apis/system.adoc

```diff
@@ -354,4 +356,59 @@ The specific contents of the JSON body are not documented here. If you choose to
 |===
 
 
+[NOTE]
+====
+This API has been available since 1.26.0
+====
+
+== Retrieve System Metrics Using Prometheus
+
+This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to setup Prometheus with the FusionAuth metrics endpoint.
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
+=== Authentication
+This endpoint requires `basic auth` authentication method. Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about `basic auth` authentication method using API keys.
+
+=== Request Parameters
+There are no request parameters required with this API.
+
+
+=== Response
+The response to this API call is various metrics that are currently available. The available metrics in this response is subject to change. The following types of gauges are currently available.
+
+* JVM Buffers - A set of gauges for the count, usage, and capacity of the JVM's direct and mapped buffer pools.
+* JVM class loading - A set of gauges for JVM classloader usage.
+* JVM garbage collection - A set of gauges for the counts and elapsed times of garbage collections.
+* JVM memory - A set of gauges for JVM memory usage, including stats on heap vs non-heap memory.
+* JVM threads - A set of gauges for the number of threads in their various states and deadlock detection.
+* Other JVM attributes - A Gauge implementation which queries an MBeanServerConnection for an attribute of an object.
+
+Here is a partial response received from this endpoint.
+
+[source,text]
```

Comment:
Please add a title and the `----` around this sample output like this to be more consistent:

```
[source,text]
.Example Prometheus Response
----
... the response below
----
```